src/query/shorthand.ts

```diff
@@ -134,12 +135,14 @@ export function fieldDef(encQ: EncodingQuery,
   // TODO: axis, legend
   for (const nestedPropParent of [Property.SCALE, Property.SORT]) {
     if (include[nestedPropParent]) {
-      if (typeof encQ[nestedPropParent] === 'string') {
-        props.push({
-          key: nestedPropParent + '',
-          value: encQ[nestedPropParent]
-        });
-      } else if (encQ[nestedPropParent] && !isEnumSpec(encQ[nestedPropParent])) {
+      if (encQ[nestedPropParent] && !isEnumSpec(encQ[nestedPropParent])) {
+        if (isString(encQ[nestedPropParent])) {
```

Comment:
Missing comment explaining what is this `isString` case?   (Can you read my comment carefully?  Again, it's not fun to keep repeating similar comments.)